components/PostAddForm.js

```diff
@@ -0,0 +1,70 @@
+import { useRouter } from 'next/router';
+import { useState, useCallback } from 'react';
+import Textarea from 'react-textarea-autosize';
+
+const PostAddForm = () => {
+  const { query: { offset } } = useRouter();
+  const [isExpanded, setExpanded] = useState(false);
+  const handleFocus = useCallback(() => setExpanded(true), []);
+  const handleCancel = useCallback(() => setExpanded(false), []);
+
+  if (offset)  {
+    return false;
+  }
+
+  return (
+    <div>
+      <Textarea
+        minRows={3}
+        maxRows={10}
+        maxLength="1500"
+        defaultValue=""
+        onFocus={handleFocus}
+      />
+      {isExpanded && (
+        <div className="actions">
+          <button className="cancelButton" onClick={handleCancel}>Cancel</button>
+          <button className="postButton">Post to my feed</button>
```

Comment:
We shouldn't use camelCase for CSS classes, and we don't need to add "button" to the class names of buttons. Also, keep in mind that the class names will be actually local to this component thanks to styled-jsx we use, so it doesn't need to be unique for the whole website. Keep it as simple as possible, only care about this particular component when styling it.